src/xrGame/RegistryFuncs.cpp

```diff
@@ -117,13 +117,13 @@ void WriteRegistry_StrValue(LPCSTR rKeyName, const char* value)
     WriteRegistryValue(rKeyName, REG_SZ, value);
 #endif
 }
-void ReadRegistry_DWValue(LPCSTR rKeyName, DWORD& value)
+void ReadRegistry_DWValue(LPCSTR rKeyName, u32& value)
```

Comment:
Uses WinAPI, should use WinAPI types.
```suggestion
void ReadRegistry_DWValue(LPCSTR rKeyName, DWORD& value)
```